variables.tf

```diff
@@ -1,51 +1,16 @@
-variable "account_id_safelist" {
-  description = "List of addtional AWS account ids (beyond the current AWS account) that the runtime can assume roles in."
-  type        = list(string)
-  default     = []
-}
-
-variable "addons" {
-  description = "List of Sym addon permissions for the runtime connector role. Addons give the runtime permissions to work with other resources without assuming another AWS role."
-  type        = list(string)
-  default     = []
-
-  validation {
-    condition     = length(setsubtract(var.addons, ["aws/secretsmgr", "aws/kinesis-firehose", "aws/kinesis-data-stream"])) == 0
-    error_message = "Unsupported addon value."
-  }
-}
-
-variable "custom_external_id" {
-  description = "The external ID to use for AWS assume role validation. If unspecified, the connector generates an external ID and the Sym platform ensures it is unique."
-  default     = ""
+variable "environment_name" {
+  description = "The unique name of the environment in which you are deploying this Sym Runtime Role. (e.g. staging, or prod)"
   type        = string
 }
 
-variable "environment" {
-  description = "An environment qualifier for the resources this module creates, to support a Terraform SDLC."
+variable "sym_account_id" {
+  description = "The AWS account ID that can assume the Sym Runtime Role. Defaults to the Sym Production AWS account ID."
   type        = string
-}
-
-variable "policy_arns" {
-  description = "Map of logical identifiers to additional IAM Managed Policy ARNs to add to the runtime connector role. The identifiers are only used for managing Terraform state."
-  type        = map(string)
-  default     = {}
-}
-
-variable "sym_account_ids" {
-  description = "List of account ids that can assume the runtime role. By default, only Sym production accounts can assume the runtime role."
-  type        = list(string)
-  default     = ["803477428605"]
+  default     = "803477428605"
```

Comment:
We could remove this variable, and use an `override.tf` when we need to change this for internal usage.

Reply:
Unfortunately, you cannot use `override.tf` on resources declared inside a module. So if we don't include this variable, we can't use this module with our staging / local setups.